src/views/OfflineDatalog.vue

```diff
@@ -1,21 +1,39 @@
 <template>
   <div class="Graph">
-    <Dropdown
-      class="channel-dropdown"
-      :options="channelsList"
-      :selected="selectedChannel"
-      :placeholder="'Select channel to plot'"
-      v-on:updateOption="onSelectedChannel"
-    >
-    </Dropdown>
-    <div class="chart-container">
-      <datalog-chart ref="datalogChart" />
-    </div>
-    <div id="container">
-      {{ offlineDatalogStatus }}
-      {{ computePacket }}
+    <ul class="bt-container">
+      <div class="sync-container">
+        <button class="sync-bt" @click="syncOfflineDatalogRecords()">
+          Sync Data
+        </button>
+      </div>
+      <div class="delete-container">
+        <button class="delete-bt" @click="$vm2.open('modal')">
```

Comment:
when calling open/close modal from template, it seems no need to ref `$vm2` just like on [this example](https://github.com/initbase/vue-modal-2#optionscomponentname)